ui/app/pages/keychains/restore-vault.js

```diff
@@ -140,16 +148,41 @@ class RestoreVaultPage extends Component {
             </div>
             <div className="import-account__input-wrapper">
               <label className="import-account__input-label">Wallet Seed</label>
-              <textarea
-                className="import-account__secret-phrase"
-                onChange={(e) => this.handleSeedPhraseChange(e.target.value)}
-                value={this.state.seedPhrase}
-                placeholder={this.context.t('separateEachWord')}
-              />
+              {showSeedPhrase ? (
+                <textarea
+                  className="import-account__secret-phrase"
+                  onChange={(e) => this.handleSeedPhraseChange(e.target.value)}
+                  value={seedPhrase}
+                  placeholder={this.context.t('separateEachWord')}
+                />
+              ) : (
+                <TextField
+                  className="import-account__textarea import-account__seedphrase"
+                  type="password"
+                  onChange={(e) => this.handleSeedPhraseChange(e.target.value)}
+                  value={seedPhrase}
+                  placeholder={t('seedPhrasePlaceholderPaste')}
+                />
```

Comment:
Not your doing originally, but why aren't both of these `TextField` components?

Also, the classnames are a bit strange to me. Why is one `__secret-phrase` and the other (i.e. new one) `__seedphrase`?

Reply:
they're not the same... one's a `textarea` (plain text) and the other is `TextField` with `type=password` (hidden)

Reply:
I just reckoned we could `TextField` for both, with `multiline=true` and no `type` prop in the plaintext case: https://material-ui.com/api/text-field/#textfield-api

@brad-decker, do we care about using the `TextField` component over e.g. a plain `textarea`?

Reply:
I care slightly less because Material-UI is of questionable longevity in our code base. That said, I tried to give this feedback on the first PR as well but stopped because I hit some complications with it. In my opinion, if you can find a way to use TextField for both -- great... but I wouldn't spend more than fifteen to thirty minutes here. Do report back with the reasons for why we can't/shouldn't use TextField with multiline=true so that the reasoning isn't lost to the void again (my bad, y'all)

Reply:
so, trying to use `TextField` and `multiline` doesn't appear to work as expected?

before: (with `textarea`)

![image](https://user-images.githubusercontent.com/675259/92000648-bd9fa180-ed0b-11ea-9a6e-4b834db3113f.png)

after: (with `TextField` and `multiline`)

![image](https://user-images.githubusercontent.com/675259/92000774-e758c880-ed0b-11ea-97fb-615e59fba2c4.png)

:woman_shrugging:

Reply:
```js
       <TextField
          id="standard-multiline-static"
          label="Multiline"
          multiline
          rows={4}
          defaultValue="Default Value"
        />
```

set Rows to 4 and maxRows to like, 10? Without max rows the text area scrolls inside of it when the text overflows.

Reply:
![image](https://user-images.githubusercontent.com/675259/92002733-2d169080-ed0e-11ea-96bf-edcee4adf6c9.png)

:woman_shrugging:

Reply:
yeah imo its not worth any further effort to get this to work appropriately. Maybe if we were adopting and embracing material-ui, but not with our current status with the library.

Reply:
Yeah, sorry for bringing up this ultimately fruitless digression 😅